.github/custom_python_actions/check_cla_pr.py

```diff
@@ -9,6 +9,7 @@
 GHIssue: TypeAlias = github3.issues.issue.Issue
 PENDING_LABEL = "cla:pending"
 APPROVED_LABEL = "cla:agreed"
+NEW_BOT_LABEL = "cla:new-bot-pending"
```

Comment:
"New" becomes not so "new" after time, but stays in the code for quite long ... Better to use something about what kind of bot it is. not that it is newer than the previous one. `cla:github-workflow-pending` or something like this maybe?